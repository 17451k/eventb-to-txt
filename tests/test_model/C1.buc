<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<org.eventb.core.contextFile org.eventb.core.comment="comment" org.eventb.core.configuration="org.eventb.core.fwd" version="3">
<org.eventb.core.extendsContext name="'" org.eventb.core.target="C0"/>
</org.eventb.core.contextFile>
