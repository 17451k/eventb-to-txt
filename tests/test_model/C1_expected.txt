context C1 // comment
    extends C0

end
